src/components/CustomerQuota/CheckoutSuccess/CheckoutSuccessCard.tsx

```diff
@@ -112,7 +106,7 @@ export const groupTransactionsByTime = (
       };
     }
     transactionsByTimeMap[transactionTimeInSeconds].transactions.push({
-      header: (policy?.name && c13nt(policy?.name)) ?? item.category,
+      header: (policy?.name && c13nt(policy?.name)) ?? c13nt(item.category),
```

Comment:
What happens when `c13nt(item.category)` returns undefined (i.e. the params were not initialised)?

Reply:
it will go to default value, which is which is in this case will return `item.category`